pkgs/development/python-modules/django-statici18n/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, django, django_appconf }:
+
+buildPythonPackage rec {
+  pname = "django-statici18n";
+  version = "2.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0cqwfirzjbanibq3mfz9lcwqnc8655zpysf9hk9g3lbwj2m478sp";
+  };
+
+  propagatedBuildInputs = [ django django_appconf ];
+
+  # pypi package does not contains test harness
```

Comment:
Can we fetch the upstream repo instead?

Reply:
I tried to setup tests from the repo as well but tests yelled at me about missing config so I did not investigate any more.